### ISSUE TITLE
Installation: Using Attachments with Pages, can't figure out where to put code

### ISSUE THREAD

**Issue by tehchriso** [jchristopher/attachments#168]
I am trying to set up Attachments on my Wordpress instance - Ver. 4.6 running the Briar theme.
I am using it as a Wiki replacement, so all of my content is saved in Pages - not Posts. 
I cannot get the attached files to show up on when viewing the Page.

I purchased the Attachments UI extension, created a new instance named 'attachments', and used the UI to generate the code snippet, but I cannot figure out which theme file I need to insert the code into.

Inserting into single.php, as shown in the Attachments UI demo video on the website, doesn't produce any results. Reading usage.md only confused me further, since it assumes you know which file The Loop is in, and as far as I can tell it only covers attachments relating to Posts - I see nothing regarding Pages.

I enabled the option for "Auto-Append" within the Attachments UI, pasted the reference code provided into the empty box, and published the instance, but it does not appear when I refresh my Page.

**Comment by tehchriso** [comment]
(Oops, posted too soon.)

For testing purposes, I've changed the theme to Twenty Fourteen and Twenty Fifteen -- it had no affect, the Attachments display does not auto-append to the Page pages.

If you could tell me where specifically within the theme files I need to insert this code so it supports Pages, not just Posts, that could be enough to get me up and running.

Or if you see anything in my screenshots that could indicate the problem, please let me know.
Thanks - and sorry for this novice question - I'm a hardware tech, anything Web is on the outer limits of my competency.

![attachmentsui settings](https://cloud.githubusercontent.com/assets/14573859/18280816/1bc06b8c-7420-11e6-9fc8-7bc3a3930180.PNG)
![files attached to post](https://cloud.githubusercontent.com/assets/14573859/18280818/1bcd0ffe-7420-11e6-925f-ac0e79b2fde4.jpg)
![attachmentsui autoappend code](https://cloud.githubusercontent.com/assets/14573859/18280817/1bca6196-7420-11e6-8cff-b51af0783895.PNG)

**Comment by tehchriso** [comment]
One more note:
We attached all of the files prior to purchasing the Attachments UI. 
So, at the time we uploaded them, I had not specifically configured any instances.

That being said, I don't *think* it's an instance issue? When I was pasting code into this php and that php file trying to figure it out on my own, I did get the data to display a couple of times (albeit breaking the theme appearance, and appearing at the very bottom of the page in raw text format).
